### PR TITLE
[pilight] Add pilight feature

### LINKED ISSUE #5742: [Pilight] Not included in any feature. Not installable via normal binding installation methods
It seems the pilight binding is not included in any of the karaf features/binding indices.
This means it cannot be installed easily via the addon UI.
I had to dig my way through the maven repo and get the the Jar file manually.

Is this intentional?

## Expected Behavior

Pilight binding is installable via addon browser from inside openHAB.

## Current Behavior

Pilight binding cannot be found in openHAB addon browser.
(The [old wiki page](https://github.com/openhab/openhab1-addons/wiki/pilight-Binding) advises to get the the addon from the download page, but it is included in none of the addon bundles)

## Possible Solution

Adding the binding to `features/openhab-addons-legacy/src/main/feature/feature.xml` may be enough. (Not an expert on karaf).

## Steps to Reproduce (for bugs)

Download the openHAB 2.4.0 distribution, addons and legacy-addons from the website.
The pilight binding is not included.
Search for the pilight binding from the addon page, it is not found.

### PR DESCRIPTION
**Pull request by idserda**
This PR adds the pilight binding to the list of features so it can be easily installed. This fixes #5742 .

### CODE REVIEW

File: features/openhab-addons-external/pom.xml [hunk 1]
```diff
@@ -108,6 +108,7 @@
                 <artifact><file>src/main/resources/conf/panasonictv.cfg</file><type>cfg</type><classifier>panasonictv</classifier></artifact>
                 <artifact><file>src/main/resources/conf/pebble.cfg</file><type>cfg</type><classifier>pebble</classifier></artifact>
                 <artifact><file>src/main/resources/conf/piface.cfg</file><type>cfg</type><classifier>piface</classifier></artifact>
+                <artifact><file>src/main/resources/conf/pilight.cfg</file><type>cfg</type><classifier>piface</classifier></artifact>
```
Comment (9037568) [inline features/openhab-addons-external/pom.xml]: This appears to be causing the build to fail. Copy/Paste error?

````
13:40:15 [ERROR] Failed to execute goal org.codehaus.mojo:build-helper-maven-plugin:1.9.1:attach-artifact (attach-artifact) on project openhab-addons-external: The artifact with same type and classifier: cfg:piface is used more than once. -> [Help 1]
````

https://ci.openhab.org/job/PR-openHAB1-Addons/2415/console

File: features/openhab-addons-external/src/main/resources/conf/pilight.cfg [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Multiple pilight instances are supported. Both the .host and .port values are required for each instance.
+# 
+# IP address of the pilight daemon
+# <instance>.host=<ip>
+#
+# Port of the pilight daemon. You must explicitly configure the port in pilight or otherwise a random port will be used and the binding will not be able to connect. 
+# <instance>.port=<port>
+#
+# Optional delay (in millisecond) between consecutive commands. Recommended value without band pass filter: 1000. Recommended value with band pass filter: somewhere between 200-500
```
Comment (9037568) [inline features/openhab-addons-external/src/main/resources/conf/pilight.cfg]: "milliseconds"

**Comment by idserda** [comment]
Thanks! Changed the classifier and the typo. Build is succeeding now.

**Comment by 9037568** [comment]
Thanks, @idserda !

**Pull request merged**